tests/src/TestBCTournament.cpp

```diff
@@ -33,12 +55,24 @@ TEST_F(TestBCTournament, GetTournamentStatus)
 	LeaveTournament();
 }
 
+TEST_F(TestBCTournament, JoinDivision)
+{
+	JoinDivision();
+	LeaveDivisionInstance();
+}
+
 TEST_F(TestBCTournament, JoinTournament)
 {
 	JoinTournament();
 	LeaveTournament();
 }
 
+TEST_F(TestBCTournament, LeaveDivisionInstance)
```

Comment:
You can remove this test. It is tested in all other tests. + it's a duplicate of JoinDivisionTest